virtool_core/models/reference.py

```diff
@@ -83,13 +83,13 @@ class ReferenceMinimal(AnalysisReference):
     data_type: ReferenceDataType
     groups: List[ReferenceUser]
     installed: ReferenceInstalled = None
-    internal_control: str
-    latest_build: ReferenceBuild
+    internal_control: str = None
+    latest_build: ReferenceBuild = None
```

Comment:
We should not use default values in models unless totally necessary. It makes things less explicit and well-defined.
```suggestion
    latest_build: Optional[ReferenceBuild]
```